site/guide/model-inventory/archive-delete-models.qmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Archive and delete models"
+date: last-modified
+---
+
+Archive models within the model inventory when they are decommissioned or delete models that you no longer require records of, keeping your inventory accurate and up to date with your organization's current resources.
+
+::: {.prereq}
+
+## Prerequisites
+
+- [x] {{< var link.login >}}
+- [x] There are models registered in the model inventory.[^1]
+- [x] You are a `Customer Admin` or assigned another role with sufficient permissions to perform the tasks in this guide.[^2]
+
+:::
+
+## Change model stage
+
+The model inventory within the {{< var validmind.platform >}} should reflect the state of models within your organization: 
+
+- If you decomission a model, you can archive it within the model inventory. 
+- Conversely, if a model is retrieved from retirement, you can activate it again.
```

Comment:
Is "retirement" a reference to some business process? If so, I wonder if we can rephrase this slightly:

```suggestion
- Conversely, if there is a renewed business need for a model that has been archived, you can activate it again.
```